portfolio/src/main/webapp/script.js

```diff
@@ -42,57 +42,29 @@ function nextImg() {
     const images = ["images/ico1.jpg", "images/ico2.jpg", "images/fountain.jpg", "images/palm-tree.jpg", "images/ice-cream.jpeg", "images/sunset.jpg"];
 
     idx = parseInt(document.querySelector('#galleryimg').dataset.idx);
-    console.log(idx + 1)
     next_idx = (idx + 1) % images.length;
     document.querySelector('#galleryimg').src = images[next_idx];
     document.querySelector('#galleryimg').dataset.idx = next_idx;
 }
 
-/* Shows or hides the Mandelbrot fractal. */
-function selectMandelbrot() {
-    if (document.querySelector('#fractal').src != "https://upload.wikimedia.org/wikipedia/commons/a/a4/Mandelbrot_sequence_new.gif")
-    {
-        document.querySelector('#fractal').src = "https://upload.wikimedia.org/wikipedia/commons/a/a4/Mandelbrot_sequence_new.gif";
-        document.querySelector('#fractal').alt = "Mandelbrot set";
-        document.querySelector('#fractal').style.visibility = "visible";
-    }
-    else
-    {
-        hideFractal();
-    }
-}
+/* Shows or hides the selected fractal. */
+function selectFractal(idx) {
+    const fractals = ["https://upload.wikimedia.org/wikipedia/commons/a/a4/Mandelbrot_sequence_new.gif",
```

Comment:
nit: One small thing you could fix is to make fractals and alts global variables. They are constant, and instead of allocating them each time the function is called, you can just allocate them once.

By the way, when I add "nit:" in front of a PR comment, it means that it is something that is nitpicky